modules/react/button/lib/ExternalHyperlink.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {styled, createComponent, StyledType} from '@workday/canvas-kit-react/common';
+import {extLinkIcon} from '@workday/canvas-system-icons-web';
+import {SystemIcon, systemIconStyles} from '@workday/canvas-kit-react/icon';
+import {Hyperlink, HyperlinkProps} from './Hyperlink';
+
+export interface ExternalHyperlinkProps extends HyperlinkProps {
+  /**
+   * Informs a screen reader user the link will open in a new window. It is read after the link text.
+   * This value will need to be translated.
+   * @default 'Opens link in new window'
+   */
+  iconLabel?: string;
+}
+
+const iconStyles = {
+  ...systemIconStyles({fill: 'currentColor', fillHover: 'currentColor'}),
+};
+
+const Anchor = styled(Hyperlink)<ExternalHyperlinkProps & StyledType>({
+  ...iconStyles,
+  display: 'inline-flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+});
+
+const iconSize = '1em';
+
+const StyledSystemIcon = styled(SystemIcon)<StyledType>({
+  ...iconStyles,
+  width: `calc(${iconSize} - 2px)`,
```

Comment:
We need to reduce the spacing after the icon otherwise our svg adds too much space between words.